dev-docs/bidders/incrementx.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: bidder
+title: IncrementX
+description: Prebid IncrementX Bidder Adaptor
+pbjs: true
+biddercode: incrementx
```

Comment:
There's also a new flag for your bidder documentation introduced in #3694

`multiformat_supported` . Possible values are

* `will-bid-on-any` - bidder will bid on any media type
* `will-bid-on-one` - bidder will prioritize one media type and ignores the others
* `will-not-bid` - bidder will not bid at all

This helps publishers / sales houses to know up front how adapter will behave in a multi-format environment.

There are also some more flags that you can lookup in the [prebid documentation on how to add an adapter](https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter).

Every aditional information helps :smile: